demisto_sdk/commands/test_content/IntegrationsLock.py

```diff
@@ -98,10 +99,20 @@ def workflow_still_running(workflow_id: str, test_playbook) -> bool:
         return True
     else:
         try:
-            workflow_details_response = requests.get(f'https://circleci.com/api/v2/workflow/{workflow_id}',
-                                                     headers={'Accept': 'application/json'},
-                                                     auth=(CIRCLE_STATUS_TOKEN, ''))
-            workflow_details_response.raise_for_status()
+            if os.environ.get('CIRCLECI'):
```

Comment:
shouldn't the CI be checked according to the workflow?
scenario:
* Circle workflow locks an integration
* Gitlab workflow want to lock the same integration

shouldn't Gitlab query Circle?

also, need to update the function doc-string.